pkg/scheduler/framework/plugins/podtopologyspread/scoring_test.go

```diff
@@ -721,19 +721,33 @@ var (
 func BenchmarkTestDefaultEvenPodsSpreadPriority(b *testing.B) {
 	for _, tt := range tests {
 		b.Run(tt.name, func(b *testing.B) {
-			pod := st.MakePod().Name("p").Label("foo", "").
-				SpreadConstraint(1, v1.LabelHostname, v1.ScheduleAnyway, st.MakeLabelSelector().Exists("foo").Obj()).
-				SpreadConstraint(1, v1.LabelZoneFailureDomain, v1.ScheduleAnyway, st.MakeLabelSelector().Exists("foo").Obj()).Obj()
+			pod := st.MakePod().Name("p").Label("foo", "").Obj()
 			existingPods, allNodes, filteredNodes := st.MakeNodesAndPodsForEvenPodsSpread(pod.Labels, tt.existingPodsNum, tt.allNodesNum, tt.allNodesNum)
 			state := framework.NewCycleState()
 			snapshot := cache.NewSnapshot(existingPods, allNodes)
-			p := &PodTopologySpread{sharedLister: snapshot}
+			p := &PodTopologySpread{
+				sharedLister: snapshot,
+				Args: Args{
+					DefaultConstraints: []v1.TopologySpreadConstraint{
+						{MaxSkew: 1, TopologyKey: v1.LabelHostname, WhenUnsatisfiable: v1.ScheduleAnyway},
+						{MaxSkew: 1, TopologyKey: v1.LabelZoneFailureDomain, WhenUnsatisfiable: v1.ScheduleAnyway},
+					},
+				},
+			}
+			client := fake.NewSimpleClientset(
+				&v1.Service{Spec: v1.ServiceSpec{Selector: map[string]string{"foo": ""}}},
+			)
+			ctx := context.Background()
+			informerFactory := informers.NewSharedInformerFactory(client, 0)
+			p.setListers(informerFactory)
+			informerFactory.Start(ctx.Done())
+			informerFactory.WaitForCacheSync(ctx.Done())
 
 			b.ResetTimer()
 
 			for i := 0; i < b.N; i++ {
 				var gotList framework.NodeScoreList
-				status := p.PreScore(context.Background(), state, pod, filteredNodes)
+				status := p.PreScore(ctx, state, pod, filteredNodes)
```

Comment:
we should also call NormalizeScore at the end, also for the default_pod_topology_spread.

Reply:
It's already here but it wasn't in the default_pod_topology_spread. This means that we have only a 3x difference, instead of 5x.